src/converter/Main.java

```diff
@@ -113,6 +113,7 @@ private static void usage() {
                 "Flame Graph options:\n" +
                 "     --title STRING     Flame Graph title\n" +
                 "     --minwidth X       Skip frames smaller than X%\n" +
+                "     --grain X          Coarsen Flame Graph to the given grain size\n" +
```

Comment:
Can you please rebase the PR on top of master, or merge master in your branch with a merge commit so that unrelated changes do not appear in diffs?

Reply:
Never mind, it integrates correctly.